assets/js/src/segments/dynamic/form.tsx

```diff
@@ -150,21 +156,16 @@ export const Form: React.FunctionComponent<Props> = ({
             type="button"
             variant="tertiary"
             iconStart={plusIcon}
-            onClick={(e): void => {
-              e.preventDefault();
-              const filters = segment.filters;
-              filters.push({
-                segmentType: SegmentTypes.WordPressRole,
-                action: SubscriberActionTypes.WORDPRESS_ROLE,
-              });
-              updateSegment({
-                filters,
-              });
-            }}
+            onClick={(e): void => addConditionAction(e, segment, updateSegment)}
```

Comment:
Does the function `addConditionAction` really need the Event? Could we call the `preventDefault` function here and then not pass the event further?

Reply:
You're right, it's not needed in this callback, I've removed it.